packages/core/src/editor/Editor.tsx

```diff
@@ -88,7 +88,7 @@ export const Editor: React.FC<React.PropsWithChildren<Partial<Options>>> = ({
     context.actions.setOptions((editorOptions) => {
       editorOptions.enabled = options.enabled;
     });
-  }, [context, options.enabled]);
+  }, [context, options?.enabled]);
```

Comment:
Let's keep the current change

Reply:
My linter wouldn't allow me to commit with the original `useEffect` code. Not sure if my eslint is wrong, but I could force commit the original code?